app/services/submit_to_app_store/prior_authority/event_builder.rb

```diff
@@ -18,14 +18,41 @@ def payload
             id: SecureRandom.uuid,
             event_type: 'provider_updated',
             details: {
-              comment: 'TODO: Extract from latest further_information',
-              documents: [], # TODO: Extract from latest further_information
+              comment: further_information_comment,
+              documents: further_information_documents,
               corrected_info: corrected_info
             },
           }
         ]
       end
 
+      def further_information_comment
+        if further_information_supplied(@application)
+          @application.further_informations
+                      .order(:created_at).last.information_supplied
+        else
+          ''
```

Comment:
any reason this is not nil?

Reply:
no, actually this should be nil, updating now